src/Microsoft.AspNetCore.ApplicationInsights.HostingStartup/Microsoft.AspNetCore.ApplicationInsights.HostingStartup.csproj

```diff
@@ -21,4 +20,4 @@
     <PackageReference Include="Microsoft.Extensions.DiagnosticAdapter" Version="$(AspNetCoreVersion)" />
```

Comment:
I'll verify that for sure as soon as I'm back from my meeting, I started the build when I left ;).